tapestry.js

```diff
@@ -846,6 +852,10 @@ function tapestryValidateNewNode(formData, isRoot) {
             errMsg += "Please enter correct media format \n";
         }
     }
+
+    if ($("#node-description-area").val() && $("#node-description-area").val().length > 250) {
```

Comment:
Please make 250 a constant/variable declared at the top

Reply:
Made this a constant